vllm/model_executor/models/mixtral.py

```diff
@@ -220,28 +220,36 @@ def process_weights_after_loading(self):
                                          requires_grad=False)
 
     def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
-        num_tokens, hidden_size = hidden_states.shape
+        batch_size, sequence_length, hidden_size = hidden_states.shape
```

Comment:
this is HPU specific and will break on other accelerators. Please cover that with is_hpu.

Reply:
Done